lexicons/app/bsky/actor/defs.json

```diff
@@ -1,6 +1,6 @@
 {
   "lexicon": 1,
-  "id": "app.bsky.actor.ref",
+  "id": "app.bsky.actor.defs",
```

Comment:
Another thing that could be pulled up into here if we like is `app.bsky.actor.profile#view` and `app.bsky.actor.profile#viewBasic`.  I think that may have been alluded to in #597 where it notes "Move the view-defs into app.bsky.actor.defs".

Reply:
ah yup 👍